Utilities/build-script-helper.py

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env python
-
-from __future__ import print_function
+#!/usr/bin/env python3
```

Comment:
Depending on the python 3 version I'm not sure just this change is enough. It's fine for valid args but running with no args ends up with a stacktrace instead of error - I believe this is caused by the behavior of `add_subparsers`. It should now take `required=True` in Python >= 3.7.